components/search/SearchSuggestion.vue

```diff
@@ -141,7 +152,18 @@
           :to="{ name: 'series-insight' }"
           @click.native="$emit('close')">
           <div :class="loadMoreItemClassName">
-            {{ $t('search.rankings') }} <span class="info-arrow">--></span>
+            {{ $t('search.rankings') }}
+            <svg
```

Comment:
would it be possible to import these in a .svg file and then import it to avoid repetition?